runtime/util/ConfigurationSerializer.cpp

```diff
@@ -327,6 +329,36 @@ void serializeConfigurationInternal(
   }
 }
 
+void serializeConfigurations(
+    const char *filename, std::unordered_set<block *, HashBlock, KEq> results) {
+  FILE *file = fopen(filename, "w");
+  auto state = serialization_state();
+
+  auto w = writer{file, nullptr};
+  auto size = results.size();
+  if (size == 0) {
+    emitConstantSort(state.instance, "SortGeneratedTopCell");
+    emitSymbol(state.instance, "\\bottom{}", size, 1);
+  } if (size == 1) {
+    auto result = *results.begin();
+    serializeConfigurationInternal(&w, result, nullptr, false, &state);
+  } else {
+    for (auto const&subject : results) {
```

Comment:
This will fail CI; there seems to be a github outage affecting the action, but you should make sure to run clang-format locally on your code.

Reply:
You're right! My bad!